test/modules/bradc/printModStuff/foo.good

```diff
@@ -35,6 +34,7 @@ Parsing module files:
   $CHPL_HOME/modules/standard/IO.chpl
   $CHPL_HOME/modules/packages/RangeChunk.chpl
   $CHPL_HOME/modules/packages/Search.chpl
+  $CHPL_HOME/modules/standard/Reflection.chpl
```

Comment:
The compiler resolves autoCopy for everything fairly early on. Now autoCopy for arrays does a lot of stuff (zippered parallel iteration in a forall loop, e.g.) and brings in more of the module code immediately.